### ISSUE TITLE
SSO Portal Review - Accessibility Fixes

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#6954]
This ticket will be used to track the progress of fixing issues from the SSO a11y review.
See the [SSO Portal A11Y Review](https://docs.google.com/document/d/1bhi7ytkpftHNCEX103BGAS5yM1ohA_fs8UPGaOAFPT0/edit?tab=t.0#heading=h.vswwvxhyd67c) for more info.

## Issue Summary Check list (WCAG)

- [x] Skip link has a "phantom jump"
- [x] input fields not announce description text (Login, Reset password, Register.. pages)
- [x] page missing meta title (All set page)
- [x] success toast has insufficient contrast (Account page)
- [x] input missing label (Verify page)

## Additional Improvements if time (a11y usability)

- [x] input announces "invalid data" on required fields that are not yet invalid (Login, Reset password, Register.. pages)
- [x] nest header links in a nav (Account page)
- [x] left nav add current page for SPA pattern (Account page)
- [x] improve Edit mode semantics (Account page) to help make it clearer what buttons do
